.github/workflows/arc-themes-base-to-new-tag.yml

```diff
@@ -0,0 +1,63 @@
+name: Tag New Arc Theme Release
+on: 
+  workflow_dispatch:
+    inputs:
+      target_release_tag:
+        description: 'Base tag (eg, 1.18 if arc-themes-release-1.18 -> arc-themes-release-1.19)'
+        required: true
+      new_release_tag:
+        description: 'New tag (eg, 1.19 if arc-themes-release-1.18 -> arc-themes-release-1.19)'
+        required: true
+
+jobs:
+  validate_tags:
+    name: Validate release numbers
+    runs-on: ubuntu-latest
+    # ensure new and target release tags match regex: 
+    # {numbers}.{numbers} requirement per product
+    steps:
+      - name: Check target release tag name
+        id: check-target-tag
+        run: |
+          if [[ ${{ github.event.inputs.target_release_tag }} =~ ^([0-9]+)\.([0-9]+)\.?$ ]]; then
+              echo ::set-output name=match::true
+          fi
+      
+      - name: Check new release tag name
+        id: check-new-tag
+        run: |
+          if [[ ${{ github.event.inputs.new_release_tag }} =~ ^([0-9]+)\.([0-9]+)\.?$ ]]; then
+              echo ::set-output name=match::true
+          fi
+
+      - name: Validate release tag numbers
+        id: validate-release-tag-numbers
+        if: steps.check-target-tag.outputs.match != 'true' && steps.check-new-tag.outputs.match != 'true'
+        # fail if one check fails
+        # passes by default
+        # exit 0 is pass
+        run: exit 1
+
+  retag:
+    name: Tag base tag to new tag
+    runs-on: ubuntu-latest
+    # previous and any future jobs need this needs property
+    needs: validate_tags
+    env: 
+      TARGET_RELEASE_TAG: arc-themes-release-${{ github.event.inputs.target_release_tag }}
```

Comment:
Looks like the agreed naming convention is `arc-themes-release-version-X.XX` as per https://arcpublishing.atlassian.net/browse/THEMES-385

Reply:
right no worries, will change

Reply:
@matthewroach updated